services/blockchain-connector/shared/sdk/genesisBlock.js

```diff
@@ -83,9 +90,81 @@ const getGenesisConfig = async () => {
 	}
 };
 
+const getGenesisAssets = async (params = {}) => {
+	const genesisBlock = await getGenesisBlockFormatted(true);
+
+	// Return all genesis block assets if no module / subStore key present in params
+	if (!params.module && !params.subStore) return genesisBlock.assets;
+
+	const assetByModule = genesisBlock.assets.find(asset => asset.module === params.module);
+
+	// Filter data by subStore if passed in input
+	if (params.subStore) {
+		let moduleData = assetByModule.data[params.subStore];
+
+		// Return empty array otherwise if passed subStore is not present
+		if (!moduleData) return [];
+
+		// Filter module data based on limit and offset
+		if (typeof params.offset !== 'undefined' && params.limit) {
+			moduleData = moduleData.slice(params.offset, params.offset + params.limit);
+		}
+
+		return [{
+			...assetByModule,
+			data: {
+				[params.subStore]: moduleData,
+			},
+		}];
+	}
+
+	// This will only be executed when params.module is present. Return the module info if found
+	return assetByModule ? [assetByModule] : [];
+};
+
+/* Returns following structure of genesis asset data filtered by module and subStore
+{
+	subStore1: [],
+	subStore2: [],
+}
+*/
+const getGenesisAssetByModule = async (params = {}) => {
+	const genesisAssets = await getGenesisAssets(params);
+
+	if (genesisAssets && genesisAssets.length) {
+		return genesisAssets[0].data;
+	}
+	return {};
+};
```

Comment:
May be we can be update this:
```suggestion
const getGenesisAssetByModule = async (params = {}) => {
	const [genesisAssets = {}] = await getGenesisAssets(params);
	return genesisAssets.data ? genesisAssets.data : genesisAssets;
};
```

Reply:
Applied.